packages/expo-gl/src/GLUtils.ts

```diff
@@ -10,6 +10,11 @@ const MAX_STRING_LENGTH = 20;
  * Sets up `__expoSetLogging` method providing some logging options useful when debugging GL calls.
  */
 export function configureLogging(gl: ExpoWebGLRenderingContext): void {
+  if ((gl as any).__expoSetLogging) {
+    // currently unnecessary, but if we move logger implementation to c++
+    // we could to that without introducing breaking changes.
+    return;
+  }
```

Comment:
```suggestion
```

I think there is no need to do this in C++ — this option is not documented and is not a part of WebGL spec, so I guess no one uses that, maybe just someone from the team for debugging purposes.